tests/docs/partials/too_large/test_payload_too_large.rst

```diff
@@ -0,0 +1,11 @@
+Scenario: An API consumer submitting a request with a request body containing 45,000 messages between 6MB and 10MB receives a 413 status code and response body containing an instance of that error
+====================================================================================================================================================================================================
+
+| **Given** the API consumer provides a message body of 45,000 messages between 6MB and 10MB
+| **When** the request is submitted
+| **Then** the response is a 413 Request Entity Too Large
+| **And** the response body contains 1 error
+
+**Asserts**
+- Response returns a 413 'Invalid Value' status code
```

Comment:
```suggestion
- Response returns a 413 'Request Entity Too Large' status code
```

Reply:
This helped me realise that the documentation didn't line up with the reality. I've now removed the `<ReasonPhrase>` thingy from the policy so that we get the apigee default of "Request Entity Too Large" in the case of 413 errors due to the payload size being between 6MB and 10MB, as well as updating this documentation.

```
< HTTP/1.1 413 Request Entity Too Large
< Date: Wed, 02 Oct 2024 13:20:11 GMT
< Content-Type: application/vnd.api+json
< Content-Length: 787
< Connection: keep-alive
< Cache-Control: no-cache, no-store, must-revalidate
< X-Content-Type-Options: nosniff
< X-Correlation-Id: rrt-8891443644323628717-c-geu2-25656-54078-1
< Strict-Transport-Security: max-age=31536000; includeSubDomains
<
```